lean/main/syntax.lean

```diff
@@ -551,4 +553,71 @@ it in the way we would expect it.
 As to how one might extend this notation to allow more set-theoretic
 things such as `{x ∈ X | p x}` and leave out the parentheses around
 the bound variables, we refer the reader to the macro chapter.
+
+
+## Exercises
+
+1. Create an "urgent minus 💀" notation such that `5 * 8 💀 4` returns `20`, and `8 💀 6 💀 1` returns `3`.
+
+**a)** Using `notation` command.  
+**b)** Using `infix` command.  
+**c)** Using `syntax` command.  
+
+Hint: multiplication in Lean 4 is defined as `infixl:70 " * " => HMul.hMul`.
+
+2. Consider the following syntax categories: `term`, `command`, `tactic`; and 3 syntax rules given below. Make use of each of these newly defined syntaxes.
+
+```
+  syntax "good morning" : term
+  syntax "hello" : command
+  syntax "yellow" : tactic
+```
+
+3. Create a `syntax` rule that would accept the following commands:
+
+- `red red red 4`
+- `blue 7`
+- `blue blue blue blue blue 18`
+
+Use the following code template:
+
+```
+syntax (name := colors) ...
+-- our "elaboration function" that infuses syntax with semantics
+@[command_elab colors] def elabColors : CommandElab := λ stx => Lean.logInfo "success!"
+```
+
+4. Mathlib has a `#help option` command that displays all options available in the current environment, and their descriptions. `#help option pp.r` will display all options starting with a "pp.r" substring.
```

Comment:
I find the task to be a little ambiguous here, people could think you want them to only  allow identifiers of options which is not possible without an elaborator, maybe make this a little more clear?